src/test/java/feature/uk/gov/ida/verifyserviceprovider/configuration/ApplicationConfigurationFeatureTests.java

```diff
@@ -54,8 +54,6 @@ public void applicationShouldStartUp() throws Exception {
             put("PORT", "50555");
```

Comment:
Rather than the MSA metadata being removed from all of these tests, I'd like to see at least one test which shows the VSP can be started with MSA metadata.  It would probably be able to share a lot of code with one of the existing tests.

Reply:
Andy and i discussed this but it seemed it would be difficult to implement so left it. I have forgotten the reason

Reply:
If it's already been considered, then fair enough.